### ISSUE TITLE
Running on folder causes nonexistent issues to be produced

### ISSUE THREAD

**Issue by rcastagno** [cflint/CFLint#463]
Attached files, if cflinted singularly produced expected result, but when evaluated together produced a bogus error:

```
  <issue severity="WARNING" id="EXCESSIVE_ARGUMENTS" message="EXCESSIVE_ARGUMENTS" category="CFLint" abbrev="EA">
    <location file="C:\temp\tesisquare\vcs\cores\framework\ttt\test_something.cfc" fileName="test_something.cfc" function="" column="0" line="3" message="Function  has too many arguments. Should be less than 10." variable="">
      <Expression><![CDATA[<cfcomponent name="another">&#x000D;&#x000A;&#x000D;&#x000A;</cfcomponent>]]></Expression>
    </location>
  </issue>
```

apparantly due to some kind of interference between the test() function of the "b" component and the component named test_something.cfc.

renamig either the function or the test_something component removing the "interference" makes the test pass

removing the commented <cfargument in the b component makes the error go away

[b.cfc.txt](https://github.com/cflint/CFLint/files/1262985/b.cfc.txt)
[test_something.cfc.txt](https://github.com/cflint/CFLint/files/1262986/test_something.cfc.txt)

**Comment by rcastagno** [comment]
This issue is blocking on SonarQube coldfusion plugin because it uses xml report to create issues on SonarQube. The existence of bogus errors causes the plugin to abruptly crash because it tries to reference a non-existent line in the file.

**Comment by ryaneberly** [comment]
@rcastagno.  Thanks for the examples.    I can't duplicate the issue in the latest dev branch.  Can this be closed?

**Comment by rcastagno** [comment]
Ok for me, will try again on 1.3 when it's out, so far I've managed to proceed on sonarqube with a workaround on the plugin side.

**Comment by ryaneberly** [comment]
Thanks @rcastagno.  Closing for now.  Reopen if it persists past 1.3.0
